### DISCUSSION TITLE
BT-775-Lot: two different codelist names in fields.json

### DISCUSSION THREAD

**Discussion by gabrieledall** [OP-TED/eForms-SDK#100] [Q&A] [answered]
The codelist referenced in BT-775-Lot has two diferent names:

"xpathRelative" : "cac:ProcurementAdditionalType/cbc:ProcurementTypeCode[**@listName='social-procurement'**]",

and 
"codeList" : {..."id" : "social-objective",...}

will these 1. remain different or will 2. both codelist names be made equal? If 2. what will be the final name?

**Comment by jpmckinney** [comment]
I see no [authority table](https://op.europa.eu/en/web/eu-vocabularies/authority-tables) for social-procurement so it must be a typo.

**Comment by gabrieledall** [comment]
Thank you for your answer. Will this typo be correted with the next SDK, then?

**Comment by pdonohoe** [comment, marked as answer]
Thank you for spotting this typo. Yes it will be corrected in the next release of the SDK.
